boxsdk/object/chunked_upload.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import unicode_literals, absolute_import
+
+import hashlib
+
+
+class ChunkedUpload(object):
+    _item_type = 'chunked_upload'
+    _uploaded_part = None
+    _part_array = []
+    _sha1 = hashlib.sha1()
```

Comment:
`_part_array` and `_sha1` need to be moved inside of `__init__()`, otherwise they'll be shared between multiple `ChunkedUpload` instances